### PR TITLE
Feature/219810 and 219811 - branding changes and hide of popup

### PR DESCRIPTION
**Pull request by sppanday**
User stories: - 219810 and 219811
=================================
This pull request consists of branding configuration:
- Config values for default color codes
- Config values for hiding pre login screen
- Config values for hiding post login prompt
- Hide "Liberate your communication" message once user is authenticated
===============================================
Images and theme style files need to be replaced with docker volume with Linux command in order to reflect changes from =========================================
aether-host-files/common/element-web/images
aether-host-files/common/element-web/styles

### CODE REVIEW

File: config-communicate.json [hunk 1]
```diff
@@ -111,9 +120,79 @@
...
     },
-    "jitsi": {
-        "preferredDomain": "jitsi.riot.im"
-    }
-}
\ No newline at end of file
+    "DISABLED_jitsi": {
+        "preferredDomain": null
+    },
+    "terms_and_conditions_links": [
```
Comment (jrschremmer) [inline config-communicate.json:132]: Please remove user settings changes in this file

File: src/components/views/auth/VectorAuthHeaderLogo.js [hunk 2]
```diff
@@ -29,13 +29,15 @@ export default class VectorAuthHeaderLogo extends React.PureComponent {
     render() {
         const brandingConfig = SdkConfig.get().branding;
         let logoUrl = "themes/element/img/logos/element-logo.svg";
+        let logoUrlSecondaryVertical = "themes/element/img/logos/element-logo-secondary-vertical.png";
```
Comment (jrschremmer) [inline src/components/views/auth/VectorAuthHeaderLogo.js:32]: Based on your conditional code below, I assume this meant to come from the config.json file?

Comment (sppanday) [reply src/components/views/auth/VectorAuthHeaderLogo.js:32]: Yes if file exist then only we will load logo otherwise element's default wont be affected.

Comment (jrschremmer) [reply src/components/views/auth/VectorAuthHeaderLogo.js:32]: Is
```
{logoUrlSecondaryVertical && 
```
a file exists check?

File: config-communicate.json (hunk 1 above)
Comment (sppanday) [reply config-communicate.json:132]: This action has been taken.

File: src/components/views/auth/VectorAuthHeaderLogo.js (hunk 2 above)
Comment (sppanday) [reply src/components/views/auth/VectorAuthHeaderLogo.js:32]: yes.

**Review by jrschremmer — APPROVED** [review]

**Pull request merged**